app/controllers/tickets_controller.rb

```diff
@@ -22,8 +22,10 @@ def show
   end
 
   def update
+    old_ticket = {title: @ticket.title.dup, description: @ticket.description.dup}
     if @ticket.update(update_params)
       render json: TicketBlueprint.render(@ticket)
+      UserMailer.with(user: @ticket.worker.user, editor: current_user.worker, old_ticket: old_ticket, new_ticket: @ticket).task_changed.deliver_later
```

Comment:
move sending email logic in model to the
`after_update or after_commit` callback (find out the difference)